not-to-release/sources/reviews/241739.xml.conllu

```diff
@@ -59,7 +59,7 @@
 15	pay	pay	VERB	VB	VerbForm=Inf	4	advcl	4:advcl:because	_
 16	attention	attention	NOUN	NN	Number=Sing	15	obj	15:obj	_
 17	to	to	ADP	IN	_	18	case	18:case	_
-18	detail	detail	NOUN	NN	Number=Sing	16	nmod	16:nmod:to	SpaceAfter=No
+18	detail	detail	NOUN	NN	Number=Sing	15	<span class="x x-first x-last">obl</span>	15:obl:to	SpaceAfter=No
```

Comment:
It looks like the beta-quality GitHub feature I used to suggest a change is buggy and inserted HTML. :P

Reply:
Yeah, weird. I'm replacing these commits anyway though.